build-tools/packages/build-cli/src/library/package.ts

```diff
@@ -503,13 +503,15 @@ export interface DependencyWithRange {
  * @param interdependencyRange - The type of dependency to use on packages within the release group.
  * @param writeChanges - If true, save changes to packages to disk.
  * @param log - A logger to use.
+ * @param onlyUpdateWorkspaceDeps - If true, only dependencies that use the workspace: protocol will be updated.
```

Comment:
Seems odd to me that the code logic is the inverse of the command flags - but it's not a big deal.

Reply:
I debated with myself a bit about it. First I decided for this partly so the flag is a bit clearer about the "special case" (only workspace deps VS all deps). Although admittedly, that still leaves the question of "what exactly is the other case", because it's probably not obvious that it is "all deps", and that applies both ways. But more importantly, I hope this flag proves unnecessary soon and we just revert most of this PR, just keeping the new functionality.